pages/partials/instructorInfoPanel.ejs

```diff
@@ -1,4 +1,4 @@
-<% if (authz_data.has_instructor_view) { %>
+<% if (authz_data.has_instructor_view || locals.is_administrator) { %>
```

Comment:
I'm a bit perplexed about why an administrator user doesn't automatically have instructor view?

Reply:
So that's the real bug? Where to start debugging that?

Reply:
Ah, ok, now I understand more.

The problem is that `has_instructor_view` is only associated with course _instances_ (because "Instructor" is a course-instance-level role, not a course role). See https://github.com/PrairieLearn/PrairieLearn/blob/master/middlewares/authzCourseInstance.js#L47 to see where `has_instructor_view` is set from the course instance permissions.

The problem with the current bug is that we don't have an active course _instance_, only an active course. The corresponding _course_ permission is `has_course_permission_view`.

So I think the correct fix is to use `authz_data.has_instructor_view || authz_data.has_course_permission_view` above.

This complicated mess is something that @tbretl's work on the new permissions system should clean up, but for now we need to survive with the janky course/course-instance distinction.

Reply:
Wait... just realized my admin check only makes sense when running locally 🤦‍♂